services/cd-service/pkg/service/batch.go

```diff
@@ -408,17 +429,32 @@ func (d *BatchServer) ProcessBatch(
 	if requiresIsolation { // This solution is not scalable and doesn't work when we have multiple cd-service pods. Ref: SRX-8JRR7Q
```

Comment:
We can move this comment to the line 433. Also, we can remove the reference to the story.

Reply:
done